src/main/java/com/direwolf20/mininggadgets/common/items/MiningGadget.java

```diff
@@ -60,6 +58,20 @@ public MiningGadget() {
         this.energyCapacity = Config.MININGGADGET_MAXPOWER.get();
     }
 
+    @Override
```

Comment:
This isn't needed, i get what you’re doing but I’d prefer it was shown as empty in the group

Reply:
this adds both the empty and full powered version of the gadget btw (removed that code to add both in newest push)